.ci/integration_test.sh

```diff
@@ -0,0 +1,240 @@
+# !/bin/bash
+
+set -e
+
+macOS=
+GOPATH_BIN=$(go env GOPATH)/bin
+SHELL_FOLDER=$(
+    cd $(dirname $0)
+    pwd
+)
+
+LOG_ERROR() {
+    content=${1}
+    echo -e "\033[31m${content}\033[0m"
+}
+
+LOG_INFO() {
+    content=${1}
+    echo -e "\033[32m${content}\033[0m"
+}
+
+execute_cmd() {
+    command="${1}"
+    eval ${command}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5569463)